internal/command/meta_backend_migrate.go

```diff
@@ -438,7 +438,10 @@ func (m *Meta) backendMigrateState_s_s(opts *backendMigrateOpts) error {
 		return fmt.Errorf(strings.TrimSpace(errBackendStateCopy),
 			opts.SourceType, opts.DestinationType, err)
 	}
-	if err := destinationState.PersistState(); err != nil {
+	// Fetching schemas during init might be more of a hassle than we want to attempt
+	// in the case that we're migrating to TFC backend, the initial JSON state won't
+	// be generated and stored.
```

Comment:
In case it matters for future ideas, the backend is currently handled before providers are installed during `init`, so requiring schemas here could lead to a catch-22 where it requires some manual intervention to proceed far enough for provider installation.

Reply:
Perhaps the "hassle" wording you used could evolve into this observation. Useful insight